### PR TITLE
Update sensor

### PR DESCRIPTION
**Pull request by golles**
Hi @Sholofly 

Nice idea for adding an extra sensor. I didn't like that it uses the standard sensor icon and I found a few copy paste thingies in the comments :-)

### CODE REVIEW

**Comment by golles** [comment]
There is one more thing I would really like to be added here, but for that, I do need your help.
I want this block to be in `sensor.py` as well, so the sensor and the media player are shown on the device page in the HA UI.

https://github.com/Sholofly/arrisdcx960/blob/dev/custom_components/arris_dcx960/media_player.py#L146-L157

In my components (which I started from a template), there is an Entity class that is extended by all entities: https://github.com/golles/ha-knmi/blob/main/custom_components/knmi/entity.py maybe this is also a way to do this here? (but as this is a technical discussion it's up to you how you want it)

File: custom_components/arris_dcx960/sensor.py [hunk 1]
```diff
@@ -38,8 +38,11 @@ def unique_id(self):
 
     @property
     def name(self):
-        """Return the unique id."""
         return f"{self.username} Recording capacity"
+    
+    @property
+    def icon(self):
+        return "mdi:percent"
```
Comment (golles) [inline custom_components/arris_dcx960/sensor.py:45]: Or:
```suggestion
        return "mdi:percent-outline"
```
Accept this if you like this better

**Comment by Sholofly** [comment]
@golles Thanks for your contribution. Good suggestion for the icon! The copy paste bugs were too obvious 'ey?
About the device_info:  I am not too familiar what the habits are for that property. In fact: your account with your provider is the integration, the media_players are part of the integration and do have a name, brand and model but the sensor doesn't have that properties. I don't know what the properties for the sensor should be or what the common properties between the players and the sensor. Do you have a suggestion?

**Comment by golles** [comment]
If the `device_info` for both the media player and the sensor are the same, they are considered as the same device and they will be shown together on the device page.

But I do think my assumption is maybe not correct, looking at the code, an account can have multiple media boxes and in such a case this new sensor is not part of a media box, but from the account in general?

**Comment by Sholofly** [comment]
That's exactly what I meant. So does that mean the mediaboxes should share the same device_info and the sensor it's own info?

**Comment by golles** [comment]
> That's exactly what I meant. So does that mean the mediaboxes should share the same device_info and the sensor it's own info?

Every media box should have its own `device_info`, so how you have it now is perfect. My additional thought about the new sensor is wrong :-)

**Pull request merged**